### PR TITLE
=typo Change all @deprecate descriptions start with upper case words.

### PR DESCRIPTION
**Pull request by He-Pin**
Just as it is.

### CODE REVIEW

File: src/library/scala/Byte.scala [hunk 1]
```diff
@@ -59,7 +59,7 @@ final abstract class Byte private extends AnyVal {
   *         filling in the new right bits with zeroes.
   * @example {{{ 6 << 3 == 48 // in binary: 0110 << 3 == 0110000 }}}
   */
-  @deprecated("shifting a value by a `Long` argument is deprecated (except when the value is a `Long`).\nCall `toInt` on the argument to maintain the current behavior and avoid the deprecation warning.", "2.12.7")
+  @deprecated("Shifting a value by a `Long` argument is deprecated (except when the value is a `Long`).\nCall `toInt` on the argument to maintain the current behavior and avoid the deprecation warning.", "2.12.7")
```
Comment (NthPortal) [inline src/library/scala/Byte.scala:62]: The files for primitive types are generated - you need to change `project/GenerateAnyVals.scala`

**Comment by hrhino** [comment]
cf. scala/scala@5562e1a

**Comment by He-Pin** [comment]
So they are lower cased by perpes ？

**Comment by He-Pin** [comment]
> cf. [5562e1a](https://github.com/scala/scala/commit/5562e1a)

Thanks, But I still think they should be uppercase like how we wrote in English

**Comment by hrhino** [comment]
I've got no strong feelings here. That PR (#5076) doesn't seem to have any explicit rationale for the change, but my guess is this:

```scala
scala> 1.round
<console>:12: warning: method round in class RichInt is deprecated (since 2.11.0): this is an integer type; there is no reason to round it.  Perhaps you meant to call this on a floating-point value?
       1.round
         ^
```

With this change it would say

```scala
scala> 1.round
<console>:12: warning: method round in class RichInt is deprecated (since 2.11.0): This is an integer type; there is no reason to round it.  Perhaps you meant to call this on a floating-point value?
       1.round
         ^
```

which has a capital letter after a colon. Maybe someone who owns a copy of [Strunk and White](https://en.wikipedia.org/wiki/The_Elements_of_Style) can chime in here.

**Comment by SethTisue** [comment]
I don’t see this as a clear improvement. The lower case letters seem perfectly fine to me.

And anyway, cosmetic changes like this that touch large numbers of files clutter up the git logs, git blame, and so forth.

Thanks Harrison for locating the previous history on this. And thanks @hepin1989 for the well-intentioned PR. But I don’t think we should merge this.

**Comment by SethTisue** [comment]
I don’t see this as a clear improvement. The lower case letters seem perfectly fine to me.

And anyway, cosmetic changes like this that touch large numbers of files clutter up the git logs, git blame, and so forth.

Thanks Harrison for locating the previous history on this. And thanks @hepin1989 for the well-intentioned PR. But I don’t think we should merge this.

**Comment by He-Pin** [comment]
Thanks ,sounds reasonable.

**Pull request closed without merging**